### ISSUE TITLE
Incorrect templating in Server config

### ISSUE THREAD

**Issue by morgante** [forseti-security/terraform-google-forseti#193]
The server config has some variables which don't get properly templated.

Ex.

```
        - resource: groups_settings_violations
          should_notify: true
          notifiers:
            
            # Upload violations to GCS.
            - name: gcs_violations
              configuration:
                data_format: csv
                # gcs_path should begin with "gs://"
                gcs_path: gs://{FORSETI_BUCKET}/scanner_violations
```

**Comment by umairidris** [comment]
I am still seeing issues with templating despite the fix. 

Error:

> yaml.scanner.ScannerError: while scanning for the next token found character '%' that cannot start any token in "/home/ubuntu/forseti-security/configs/forseti_conf_server.yaml", line 269, column 5
> at fetch_more_tokens (/usr/local/lib/python3.6/dist-packages/yaml/scanner.py:260)
> at check_token (/usr/local/lib/python3.6/dist-packages/yaml/scanner.py:116)
> at parse_block_mapping_value (/usr/local/lib/python3.6/dist-packages/yaml/parser.py:449)
> at check_event (/usr/local/lib/python3.6/dist-packages/yaml/parser.py:98)
> at compose_node (/usr/local/lib/python3.6/dist-packages/yaml/composer.py:64)
> at compose_mapping_node (/usr/local/lib/python3.6/dist-packages/yaml/composer.py:133)
> at compose_node (/usr/local/lib/python3.6/dist-packages/yaml/composer.py:84)
> at compose_document (/usr/local/lib/python3.6/dist-packages/yaml/composer.py:55)
> at get_single_node (/usr/local/lib/python3.6/dist-packages/yaml/composer.py:36)
> at get_single_data (/usr/local/lib/python3.6/dist-packages/yaml/constructor.py:35)
> at load (/usr/local/lib/python3.6/dist-packages/yaml/__init__.py:72)
> at safe_load (/usr/local/lib/python3.6/dist-packages/yaml/__init__.py:94)
> at _parse_yaml (/usr/local/lib/python3.6/dist-packages/forseti_security-2.17.0-py3.6.egg/google/cloud/forseti/common/util/file_loader.py:192)

At line 269 I see:

>     # Provide connector details
>     %{ if SENDGRID_API_KEY != "" }
>     email_connector:
>       name: sendgrid
>       auth:
>         api_key: 
>       sender: 
>       recipient: 
>       data_format: csv
>     %{ endif }

In fact, none of the %{ lines seem to be parsed correctly.

"terraform_version": "0.11.14"

**Comment by morgante** [comment]
@umairidris I'm unable to reproduce thus far (on `master)`. if you check the config file in the GCS bucket, does it also have the same issue?

**Comment by umairidris** [comment]
After some digging this is due to an outdated template provider. The support for %{} syntax is added in v2.0. Feel free to close this, thanks!

**Comment by morgante** [comment]
Thanks for catching that!

I opened https://github.com/forseti-security/terraform-google-forseti/pull/195 to track the requirement.

**Comment by aaron-lane** [comment]
Fixed by #194.

**Comment by morgante** [comment]
@aaron-lane Please leave this open until you have reviewed and merged #195.
